### PR TITLE
Replaced local VncConsole by patternfly VncConsole

### LINKED ISSUE #1134: Replaced local VncConsole by patternfly VncConsole
We had VncConsole in repository because PF3 console didn't had needed functionality, but now PF3 VncConsole have everything that needed, so we can remove component in our repository and replace it by component from PF3.

### PR DESCRIPTION
**Pull request by bond95**
Fixes #1134

### CODE REVIEW

**Comment by sjd78** [comment]
ci test please

**Comment by bond95** [comment]
ci test please

**Comment by sjd78** [comment]
@bond95 - CI is failing here because `jest`'s dependencies jumped up quite a few versions in the yarn.lock file.  Maybe try reverting your `yarn.lock` file doing a new `yarn install` and seeing if the jest versions get bumped up again?  If that doesn't work, I recommend just updating to a new(er) version of jest with this change and adjusting the jest configs in `package.json` accordingly.

**Comment by sjd78** [comment]
ci test please

**Comment by sjd78** [comment]
ci test please

File: config/webpack.config.commons.js [hunk 1]
```diff
@@ -24,6 +24,7 @@ exports.addFlowBabelWebpackPlugin = function (webpackConfig) {
 
 exports.commonAliases = function () {
   return {
+    '@spice-project/spice-html5': `${paths.appSrc}/emptyFile.js`,
```
Comment (sjd78) [inline config/webpack.config.commons.js]: Why is this necessary?  It would be nice to have a comment in the code to explain.

File: src/components/VmConsole/index.js [hunk 2]
```diff
@@ -5,11 +5,12 @@ import { push } from 'connected-react-router'
 import $ from 'jquery'
 
 import style from './style.css'
-import VncConsole from '_/react-console'
+// import VncConsole from '_/react-console'
```
Comment (sjd78) [inline src/components/VmConsole/index.js]: just remove the commented line!

File: src/components/VmConsole/index.js [hunk 3]
```diff
@@ -111,8 +112,8 @@ class VmConsole extends React.Component {
             path={proxyTicket}
             host={websocket.get('host')}
             port={websocket.get('port')}
-            toolbarContainer='vm-console-toolbar-sendkeys'
-            containerId={NOVNC_CONTAINER_ID}
+            portalToolbarTo='vm-console-toolbar-sendkeys'
+            // containerId={NOVNC_CONTAINER_ID}
```
Comment (sjd78) [inline src/components/VmConsole/index.js]: just remove the commented line!

**Comment by sjd78** [comment]
@bond95 - This PR doesn't have an issue attached so it won't be on jira for the QE folks to pickup.  So I see either this is a developer only change (and we can just merge it) or go ahead and create an issue for it and put it in the right place on jira once the ticket gets created over there.

**Comment by sgratch** [comment]
> @bond95 - This PR doesn't have an issue attached so it won't be on jira for the QE folks to pickup. So I see either this is a developer only change (and we can just merge it) or go ahead and create an issue for it and put it in the right place on jira once the ticket gets created over there.

Done

**Comment by isaranova** [comment]
ci build please

**Comment by sgratch** [comment]
@bond95 CI failed, can you please check?

**Comment by isaranova** [comment]
ci build please

**Comment by isaranova** [comment]
CI is still failing, cannot build repo to test.

**Comment by sgratch** [comment]
@bond95  hi, can you please verify why CI failed or have the checkbox for "Allow edits from maintainers" checked?

**Comment by bond95** [comment]
Cannot be merged until https://github.com/patternfly/patternfly-react/pull/3465 is not merged

**Comment by bond95** [comment]
Updated pre-seed: https://gerrit.ovirt.org/#/c/106178/

**Comment by michalskrivanek** [comment]
> Updated pre-seed: https://gerrit.ovirt.org/#/c/106178/

@bond95 I updated that and it looks to me. I suppose it contains the patternfly fix you required, right?

**Comment by bond95** [comment]
@michalskrivanek not yet, PF4 PR is still in review.

**Comment by sjd78** [comment]
ci test

**Comment by sjd78** [comment]
https://github.com/patternfly/patternfly-react/pull/3465 Has been merged but I don't think it has been included in a release yet.  Closer!

**Comment by bond95** [comment]
According to commit history PR should be included to this release https://github.com/patternfly/patternfly-react/releases/tag/%40patternfly%2Freact-console%402.0.10

**Comment by sjd78** [comment]
> According to commit history PR should be included to this release https://github.com/patternfly/patternfly-react/releases/tag/%40patternfly%2Freact-console%402.0.10

Ok cool.  Looks like **2.0.11** is published: https://www.npmjs.com/package/@patternfly/react-console.  @bond95, can you update package.json and yarn.lock with the new version?  I can handle the nodejs-modules pre-seed for you.

**Comment by isaranova** [comment]
LGTM
